analyses/eth/docs/eth-risk-anlaysis.Rmd

```diff
@@ -0,0 +1,188 @@
+
+# Put a figure in here, for example 
+
+flood_map
+
+drought_map
+
+```
+
+The tables below show the full lists of both the flood and drought events from EM-DAT. 
+
+```{r event-table-flood, dpi=300, include=T, message= FALSE, warning=FALSE}
+
+emdat_flood %>%
+  select(c(Year, Location, Origin, `Total Affected`)) %>%
+  knitr::kable(caption = "List of flood events",
+               align = c('l')) %>%
+        collapse_rows(columns = 1, valign = "top") %>%
+        kable_styling(bootstrap_options = c("striped", "hover", "condensed", "responsive"),
+                      fixed_thead = T) %>%
+        scroll_box(width = "100%", height = "500px")
+
+```
+
+<br>
+<br>
+
+```{r event-table-drought, dpi=300, include=T, message= FALSE, warning=FALSE}
+
+emdat_drought %>%
+  select(c(Year, Location, Origin, `Total Affected`)) %>%
+  knitr::kable(caption = "List of drought events",
+               align = c('l')) %>%
+        collapse_rows(columns = 1, valign = "top") %>%
+        kable_styling(bootstrap_options = c("striped", "hover", "condensed", "responsive"),
+                      fixed_thead = T) %>%
+        scroll_box(width = "100%", height = "500px")
+
+```
+
+<br>
+<br>
+
+### Other potentially relevant datasets 
+
+1. [Global Flood Database](http://global-flood-database.cloudtostreet.info/#interactive-map)
+- Gridded data of historical flood extents available via [Google Earth Engine](https://developers.google.com/earth-engine/datasets/catalog/GLOBAL_FLOOD_DB_MODIS_EVENTS_V1).
+- Includes flood events from 2000-2018. Ethiopia has 28 events recorded.
+- Events identified from the Dartmouth Flood Observatory, with flood extent mapped using MODIS satellite imagery.
+
+
+2. [IDMC](https://data.humdata.org/dataset/idmc-idp-data-for-ethiopia)
+- Number of IDPs, relating to conflict and disaster.
+- Not subnational.
+- Data available between 2008-2020.
+
+
+3. [FAOSTAT Food Security Indicators](https://data.humdata.org/dataset/faostat-food-security-indicators-for-ethiopia)
+- National level indicators from 2000-2020
+- Such as: Prevalence of undernourishment (percent) (3-year average), Number of severely food insecure people (million) (3-year average), Percentage of population using safely managed sanitation services (Percent)
+
+## Near-present situation
+
+### IDP population
+
+The figures below show the basic spatial distribution, volume, and demographics of Ethiopia's IDP population as identified by the [IOM DTM Site Assessment](https://data.humdata.org/dataset/ethiopia-baseline-assessment-data-iom-dtm) conducted in July 2021. 
+
+```{r img1, echo = FALSE, include=T, fig.cap='IDP population breakdown by gender at Admin 2 level, symbols scaled by total IDP population', out.extra='style=padding:5px;"', fig.align='center'}
+images1 <- c('img/gender-map-pop.png')
+include_graphics(images1)
```

Comment:
Personal preference, but really don't like pie charts on their own, let alone splattered across a map. Is there anything in here we are actually wanting to highlight for the reader (this and below graphs) beyond the main areas of displacement? Also, just wondering why 1 administrative area is not coloured in? I'm assuming this comes out of a DTM report so probably unanswerable ha, just wondering.

Reply:
Although I'm not a huge fan of pie charts either I actually like this graph (by gender). It conveys a lot of information succinctly. The age breakdown is a bit messier to follow. Perhaps side-by-side graphs would clearer?